src/components/atoms/Timeline/index.tsx

```diff
@@ -35,15 +52,76 @@ const Timeline: React.FC<Props> = memo(
       strongScaleHours,
       currentPosition,
       events,
+      player: {
+        playSpeed,
+        onChangePlaySpeed,
+        formattedCurrentTime,
+        isPlayed,
+        isPlayBacked,
+        toggleIsPlayed,
+        toggleIsPlayBacked,
+      },
     } = useTimeline({
       currentTime,
       range,
       onClick,
       onDrag,
+      onPlay,
     });
+    const t = useT();
+
+    if (!isOpened) {
+      return (
+        <OpenButton onClick={onOpen}>
+          <Icon alt={t("Open timeline")} icon="timeline" size={24} />
+        </OpenButton>
+      );
+    }
 
     return (
       <Container>
+        <CloseButton themeColor={themeColor} onClick={onClose}>
+          <Icon alt={t("Close timeline")} icon="cancel" size={16} />
+        </CloseButton>
+        <ToolBox>
+          <>
```

Comment:
I can't see a reason for a fragment here, so if it isn't needed please remove it.

Reply:
This was fixed by [refactor: remove unnecessary fragment](https://github.com/reearth/reearth-web/pull/277/commits/f74dd9579b4db1d4ad4456f8e8afca3f149cf007)